### ISSUE TITLE
R command not found error

### ISSUE THREAD

**Issue by CuriousTim** [jalvesaq/Nvim-R#571]
When I open an R script file for editing and press `<LocalLeader>rf`, I get the error
```
zsh:1: command not found: R^@
```
I am using Vim version 8.2

I do have R installed and when I enter `R` in my terminal, R starts up just fine, so it is in my PATH. Is it possible that some extra characters are being appended to the end of the R invocation? Any idea what might be going on?

**Comment by jalvesaq** [comment]
What should be my settings to be able to replicate the issue? I noted that zsh is being called. Did you set it as the shell for Vim?

**Comment by CuriousTim** [comment]
I don't believe I every manually changed the shell for Vim, but my default shell is zsh. This is my `.vimrc`
```
" The vimrc
set nocompatible

set encoding=utf-8

call plug#begin('~/.vim/plugged')
Plug 'jalvesaq/Nvim-R', {'branch': 'stable'}
call plug#end()

let mapleader=","
let maplocalleader="\\"

" Easy editing of vimrc
nnoremap <leader>ev :vsplit $MYVIMRC<cr>
nnoremap <leader>sv :source $MYVIMRC<cr>

" Disable viminfo
set viminfo=

" gopass
au BufNewFile,BufRead /dev/shm/gopass.* setlocal noswapfile nobackup noundofile

" Whitespace
set tabstop=8
set softtabstop=4
set shiftwidth=4
set expandtab
set cindent

" Line numbers
set number

syntax on
filetype on
filetype plugin on
filetype indent on

"Nvim-R
let R_path="/usr/bin"
let R_app="R"
```
I set the `R_path` and `R_app` variables in an attempt to debug this issue. The problem occurred before I added those entries into my `.vimrc`

I am using oh-my-zsh so a lot of my zsh settings are modified, but I tried running everything in a interactive shell started with
```
$ zsh -df
```
which I believe ignores all my rc files and the same issue occurs.

**Comment by jalvesaq** [comment]
I'm sorry, but I can't replicate the issue. I started zsh and started Vim within the zsh session, using the stable branch of Nvim-R.

 R started with no issue.

**Comment by CuriousTim** [comment]
Yeah it's strange because I have used zsh with Nvim-R before with no problem. Hopefully the problem works itself out. Thanks anyways.

**Comment by Tongzhoutz** [comment]
Hello, have you resolved the issue? I do have the same issue. I suspect this may be related to the path of ```python3``` or ```python```

**Comment by CuriousTim** [comment]
The solution for me was to fix the `PATH` variable in my `.zshenv` file. I had prepended paths incorrectly i.e. `PATH="/foo/bar"` instead of `PATH="/foo/bar:$PATH"` so R was not available in the vim terminal emulator. I don't really understand what was going on, but it seems like the vim terminal emulator was starting zsh with a different set of startup files than my interactive login shell so I was able to access R in one, but not the other.
